core/assets/environment_variables/staging.json

```diff
@@ -1,4 +1,8 @@
 {
   "AVAILABLE_LOCALES": "en",
```

Comment:
variables like these can be abstracted to common variables that will be set for all envs at once, that way it will be easier to modify them and we will not risk to update variable only for one env and forgot to update it for another env

Reply:
It was updated in 7377537 ✅